test/helpers/deploy/deploy-zns.ts

```diff
@@ -171,45 +171,31 @@ export const deployDomainToken = async (
 export const deployMeowToken = async (
   deployer : SignerWithAddress,
   isTenderlyRun : boolean
-) : Promise<MeowTokenMock> => {
-  const factory = new MeowTokenMock__factory(deployer);
+) : Promise<ERC20Mock> => {
+  const factory = new ERC20Mock__factory(deployer);
 
-  const meowToken = await hre.upgrades.deployProxy(
-    factory,
-    [
-      meowTokenName,
-      meowTokenSymbol,
-    ],
-    {
-      kind: "transparent",
-    }
-  ) as unknown as MeowTokenMock;
+  const meowToken = await factory.deploy(
+    meowTokenName,
+    meowTokenSymbol,
+  ) as unknown as ERC20Mock;
```

Comment:
why `as unknown` here before `as ERC20Mock`? Coming from the `ERC20Mock__factory` above the type shouldn't need `unknown` first